### ISSUE TITLE
Tinkers tool script not working the way i intend

### ISSUE THREAD

**Issue by scalda** [jaredlll08/ModTweaker#473]
#### Issue Description:
Hi, I am trying to add manasteel tools/parts via MT. I am using naturals pledge as the source for the molten material. but for the life of me i can't get it to output the correct item to the correct cast.

I always get the first item (in the case of the scripts a shard) no matter what cast i use.

Is there something really stupid i am overlooking?

Regards

Scalda

#### What you expected to happen:

the correct cast to be used for the item i am trying to make

#### Script used (Please Pastebin or gist your script, posting an unpasted or ungist'd script will automatically close this issue):

[script](https://pastebin.com/YhC3Vytg)

#### Minetweaker.log file (Please Pastebin or gist your file, posting an unpasted or ungist'd file will automatically close this issue):

[log](https://pastebin.com/hNNPeDu8)
____
#### Affected Versions (Do *not* use "latest"):

- Minecraft: 1.10.2
- Forge: 12.18.3.2254
- Crafttweaker: CraftTweaker-1.10.2-3.0.24.jar
- ModTweaker: ModTweaker2-2.0.11.jar
- MTLib: MTLib-1.0.2.jar

**Comment by bookerthegeek** [comment]
I am curious as to why you need scripts to do this? This is without any scripts installed. Also, updated your forge.
![It works?][profit]


[profit]:https://i.imgur.com/26ei1Kp.png

**Comment by scalda** [comment]
i needed scripts because we couldn't craft the parts with the liquid from Natural Pledge.

As for forge i didn't know there was a newer version as thought they had abandoned 1.10.2 in favour of 1.11.2

**Comment by bookerthegeek** [comment]
What versions of Botania and Naturals Pledge are you using?

**Comment by scalda** [comment]
Botania: Botania r1.9-341.jar
Naturals Pledge: NaturalPledge-r1.11.jar

**Comment by scalda** [comment]
Well it looks like something in the version of foolcraft that we are developing is causing issues with the tinkers integration.

**Comment by bookerthegeek** [comment]
Got a link to the Dev Import for Curse that you're working with? It should be working, so lets go figure out why it is not working.

**Comment by scalda** [comment]
[Uploading FoolCraft-tinkers dev.zip…](dev pack)

**Comment by bookerthegeek** [comment]
From what I can tell it is registering the recipes to any cast. First one in the list is what is comming out. So if you put a Tool Rod first that is the output you get. Odd. Also it only affects the casts that use NBT data, but not the Ingot or Nugget cast as long as you register those first.

**Comment by bookerthegeek** [comment]
@scalda I do apologize but I could not figure out how to get the casts to work correctly, although the script is technically correct there is as you thought an integration bug between `Natural Pledge` and `PlusTic`.

That being said, [here][1] is an alternative set for you to use. It uses the Iron version of the parts, and you cover it with mana steel in the part table loosing the iron part. Hope this helps till a solution is found. Feel free to ping me if you would like some more help.

[1]:https://gist.github.com/bookerthegeek/70ee9f4800aa9f5b07ef3a9c3f9c11cc

**Comment by scalda** [comment]
@bookerthegeek Thanks for the hard work you have done here. I think we came up with a solution overnight must have been before these messages from you.

We rolled naturals pledge back to before they added their tinkers support and reenabled the botania support in plustic. and all is working except i had to make scripts for the blocks to be casted in the basin

But i will keep your script for the future as we may have to change it again

Thanks again

Scalda

I consider this issue closed
